app/src/main/AndroidManifest.xml

```diff
@@ -98,71 +101,62 @@
                 <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\.mkv" />
                 <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\..*\\.mkv" />
                 <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\..*\\..*\\.mkv" />
-
```

Comment:
fyi: pure formatting changes make this hard to review and cause merge conflicts

Reply:
i tried to revert as much as possible, but yk autoformat